sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -2489,10 +2693,6 @@ abstract class CSVSuite
   }
 
   test("SPARK-36536: use casting when datetime pattern is not set") {
-    def isLegacy: Boolean = {
```

Comment:
I think we should still test the legacy behavior for LTZ?

Reply:
Oh, this is for TimestampNTZ change - isLegacy flag was only applied to TimestampNTZ, TimestampLTZ code is unchanged.